### PR TITLE
vstest.console without arguments should print error message and help …

### PR DESCRIPTION
**Pull request by jayaranigarg**
…contents

### CODE REVIEW

**Review by singhsarab — APPROVED** [review]

File: test/vstest.console.UnitTests/ExecutorUnitTests.cs [hunk 1]
```diff
@@ -60,25 +60,17 @@ public void ExecutorPrintsSplashScreenTest()
...
             var mockOutput = new MockOutput();
             var exitCode = new Executor(mockOutput, this.mockTestPlatformEventSource.Object).Execute(null);
 
-            // Since no projectjsons exist in current folder it should fail
-            Assert.AreEqual(1, exitCode, "Exit code must be One for bad arguments");
-
-            //// Verify that messages exist
-            //Assert.IsTrue(mockOutput.Messages.Count > 0, "Executor must print atleast copyright info");
-            //Assert.IsNotNull(mockOutput.Messages.First().Message, "First Printed Message cannot be null or empty");
+            Assert.AreEqual(1, exitCode, "Exit code must be One when no arguments are provided.");
```
Comment (Faizan2304) [inline test/vstest.console.UnitTests/ExecutorUnitTests.cs]: We should also check that help content is getting shown on screen. Its seems like there is no way here to figure this out. 
I would suggest to add acceptance test for it.

**Review by Faizan2304 — CHANGES_REQUESTED** [review]

File: test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj [hunk 2]
```diff
@@ -11,6 +11,7 @@
     <AssemblyName>Microsoft.TestPlatform.AcceptanceTests</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
+    <!--<ProjectReference Include="..\..\src\vstest.console\vstest.console.csproj" />-->
```
[the remark this answers is not in the document]
Comment (Faizan2304) [reply test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj]: Please remove this line

**Review by Faizan2304 — APPROVED** [review]
LGTM 👍 J

Removed commented line from this file
https://github.com/Microsoft/vstest/pull/1110/files#diff-baa514d4de14d5a192f4b8cf8e5d7312

File: test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj [hunk 3]
```diff
@@ -11,6 +11,7 @@
     <AssemblyName>Microsoft.TestPlatform.AcceptanceTests</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\src\vstest.console\vstest.console.csproj" />
```
Comment (Faizan2304) [inline test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj]: Why do we need this?

**Pull request merged**